### ISSUE TITLE
Columns wise expand

### ISSUE THREAD

**Issue by ayankapadiya** [silevis/reactgrid#251]
Thanks for this delicious Grid with expandable functionality.

I want to perform column-wise expansion. Is there a code sample or example for this?

![image](https://github.com/silevis/reactgrid/assets/145652510/e9aefbb8-3271-4185-92c5-09c755e1b773)

**Comment by DLowHP** [comment]
Take a look at this example's [source](https://github.com/silevis/reactgrid-website/blob/gh-pages-dev/src/samples/samplesSrc/bp/BP.tsx) - especially the [lines 45-56](https://github.com/silevis/reactgrid-website/blob/5967c2bd06756e1047f82fd11659e1ac77fe728f/src/samples/samplesSrc/bp/BP.tsx#L45-L56) and the [helper functions](https://github.com/silevis/reactgrid-website/blob/gh-pages-dev/src/samples/samplesSrc/bp/helpersFunctions.ts).

**Comment by ayankapadiya** [comment]
Hey! Thanks for this. I have implemented column-wise grouping in my code. 

Need 1 help, The request call back column grouping(expand) not working.

Call back means: 
1) I have a 1 submit button to get data **(dynamic rows & columns)** from the backend.
2) The rows & cols I set up in the react grid.
3) 1st time it is working fine. but, when submitting and fetching new data and setting up in rows and cols that column grouping not working
